fem/bilinearform.hpp

```diff
@@ -337,6 +337,12 @@ class BilinearForm : public Matrix
       return *mat_e;
    }
 
+   // Returns true if the sparse matrix of eliminated b.c.s is not null, false otherwise.
```

Comment:
```suggestion
   /// @brief Returns true if the sparse matrix of eliminated b.c.s is not null,
   /// false otherwise.
   ///
   /// @sa SpMatElim().
```

Maybe the documentation for `SpMatElim` should also be changed indicating that those functions will fail if `HasSpMatElim` returns false.

Reply:
Done. I think I did it correctly wrt the doxygen references but please take a look. I also added `HasSpMat` since that will fail in the same way.